test/e2e/specs/appearance/widgets__legacy-visibility.ts

```diff
@@ -45,20 +52,15 @@ skipDescribeIf( envVariables.TEST_ON_ATOMIC )(
 
 		// @todo: Refactor/Abstract these steps into a WidgetsEditor component
 		// Skipped for mobile due to https://github.com/Automattic/wp-calypso/issues/59960
-		skipDescribeIf( envVariables.VIEWPORT_NAME === 'mobile' )(
-			'Regression: Verify that the visibility option is present',
-			function () {
-				it( 'Insert a Legacy Widget', async function () {
-					await page.click( 'button[aria-label="Add block"]' );
-					await page.fill( 'input[placeholder="Search"]', 'Top Posts and Pages' );
-					await page.click( 'button.editor-block-list-item-legacy-widget\\/top-posts' );
-				} );
+		it( 'Insert a Legacy Widget', async function () {
```

Comment:
The test hierarchy has been flattened here as the skip clause has been moved up to the top-level `describe`.